bigbluebutton-html5/imports/api/chat/server/publishers.js

```diff
@@ -6,7 +6,7 @@ import { isAllowedTo } from '/imports/startup/server/userPermissions';
 
 Meteor.publish('chat', (credentials) => {
   if (!isAllowedTo('subscribeChat', credentials)) {
-    this.error(new Meteor.Error(402, "The user was not authorized to subscribe for 'chats'"));
+    throw new Meteor.Error(402, "The user was not authorized to subscribe for 'chats'");
```

Comment:
Any idea of the differences between `this.error` and `throw`? I fixed those same issues changing from arrow functions to default functions expressions.